spec/support/shared_examples/merge_time_distribution_data.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.shared_examples 'merge time data distribution' do
+  it 'returns an array' do
+    expect(subject).to be_an(Array)
+  end
+
+  context 'when name is merge time' do
+    before do
+      values.each do |value|
+        pull_request = create :pull_request, project: project
+        create(:merge_time, pull_request: pull_request, value: value)
+      end
+      query.merge!(name: :merge_time)
+    end
+
+    it 'returns an array with name key' do
+      expect(subject.first).to have_key(:name)
+    end
+
+    it 'returns an array with name data' do
+      expect(subject.first).to have_key(:data)
+    end
+
+    it 'returns an array with filled value' do
+      expect(subject.first[:data].empty?).to be false
```

Comment:
Maybe we can use the [be_empty](https://relishapp.com/rspec/rspec-expectations/v/3-9/docs/built-in-matchers/predicate-matchers#should-not-be-empty-(based-on-array#empty?)) matcher